src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -850,6 +857,13 @@ function addCORSHeaders(res) {
   )
 }
 
+function createBadAuthorizationHeaderError(): void {
+  throw httpError(
```

Comment:
Don't throw here, we throw down below (and the name is just `create`)

Reply:
whoops! great catch. thx. fb7430b